Sources/MapboxNavigation/SpeedLimitView.swift

```diff
@@ -51,6 +51,13 @@ public class SpeedLimitView: UIView {
         }
     }
     
+    /**
+     The current speed to display.
```

Comment:
To avoid confusion, leave this property `:nodoc:`’d and omit `SpeedLimitView` from the changelog until we make use of it.